src/plugins/uploader.js

```diff
@@ -10,26 +10,27 @@ const readChunk = require("read-chunk");
 const crypto = require("crypto");
 const isUtf8 = require("is-utf8");
 const log = require("../log");
+const contentDisposition = require("content-disposition");
 
 // List of allowed mime types that can be rendered in browser
 // without forcing it to be downloaded
```

Comment:
This needs an updated comment.
Reading this, it's hard to see what the map is for and what the mapping is supposed to do, considering that the comment says what it says (it's not a list any longer).

Reply:
tbh I didn't even notice the comment lel, will change it